src/utils.js

```diff
@@ -718,6 +718,11 @@ export function replaceAuctionPrice(str, cpm) {
   return str.replace(/\$\{AUCTION_PRICE\}/g, cpm);
 }
 
+export function replaceClickThrough(str, clicktag) {
+  if (!str) return;
+  return str.replace(/\$\{CLICKTHROUGH\}/g, clicktag);
```

Comment:
should validate that `clicktag` is defined and not NULL, maybe that it's a string.